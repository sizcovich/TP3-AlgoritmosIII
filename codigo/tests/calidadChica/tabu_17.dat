5 15 14 823400
5 25 24 2498800
5 35 34 5145800
5 45 44 10381900
5 55 54 16691800
5 65 64 5866400
5 75 74 11884000
5 85 84 20492300
5 95 94 32381800
5 105 104 47499800
