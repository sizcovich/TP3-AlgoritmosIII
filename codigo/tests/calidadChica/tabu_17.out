5 1 9 
5 1 2 
5 1 20 
5 1 31 
5 1 51 
5 1 64 
5 1 54 
5 1 22 
5 1 24 
5 1 7 
